pkg/common/k8s_utils.go

```diff
@@ -170,16 +167,6 @@ func ObjectKeyListDifference(a, b []client.ObjectKey) []client.ObjectKey {
 	return result
 }
 
-// ContainsObjectKey tells whether a contains x
-func ContainsObjectKey(a []client.ObjectKey, x client.ObjectKey) bool {
-	for _, n := range a {
-		if x == n {
-			return true
-		}
-	}
-	return false
-}
-
```

Comment:
Removing as it's already provided by:
-  https://pkg.go.dev/golang.org/x/exp@v0.0.0-20230817173708-d852ddb80c63/slices#Contains